docusaurus/docs/engineering/start-developing/boba-backend.md

```diff
@@ -43,8 +39,8 @@ touch firebase-sdk.json
 
 ### 2 — Set up your FirebaseSDK credentials.
 
-:::warning
-**This is an INVALIDATED Firebase service account key.** It will allow the firebase admin SDK to be correctly initialized, but doesn't grant any further privilege.
+:::info
+**This is an <u>INVALIDATED</u> Firebase service account key.** It will allow the firebase admin SDK to be correctly initialized, and allow you to login with your BobaBoard credentials, but doesn't grant any further privilege.
```

Comment:
I'd omit the part about not granting further privileges. I'm not sure what it refers to but it might be the reason it's not immediately obvious that we can actually post to the frontend while running it locally

Reply:
Changeed it a bit